src/components/Button/src/Button.vue

```diff
@@ -75,7 +75,8 @@ function setColorVariables(tokens, variant) {
 	const hoverStateAdjust = 0.08;
 	const activeStateAdjust = 0.16;
 	const focusAlphaAdjust = 0.3;
-	const colorHover = colorMainHover[stateAdjustment](hoverStateAdjust).toHex();
+	const colorHover = colorMainHover === colorMain
```

Comment:
this works but I wonder if it might be better to refactor all the hover logic so that if we've already got `tokens.colorHover`, we immediately assign `colorHover` to that value, and we skip calculating the `stateAdjustment` and `hoverStateAdjust` variables.

Reply:
Updated so that we pass `tokens.colorHover` for hover color but we still need to calculate some of the `stateAdjustment` values for the active and focus states.